fontc_crater/README.md

```diff
@@ -22,14 +22,17 @@ script in the [`fontc_crater` repo][crater-repo] and results are posted to
 To run in CI mode locally to play with the html output:
 
 When run in CI, we want to ensure that we have complete control over the set of
-python pacakges we are comparing against; to ensure this we use `constraints.txt`
-in the root `resources/scripts` directory. To update the set of python packages
-in use, you need to update this file:
+python pacakges we are comparing against; to ensure this we use `pip-compile`
+(part of [`pip-tools`]) to generate a pinned `requirements.txt` from a
+`requirements.in` file in the root `resources/scripts` directory. To update the
+set of python packages in use, you need to update this file:
 
 - `rm -rf` your existing venv
 - create a new venv and activate it
```

Comment:
one doesn't need to delete and create a new venv when using pip-compile, it may well run from a global site-packages (e.g. I install it globally with `pipx install pip-tools`) as pip-compile does not modify its environment.

Reply:
installing something in a venv is generally good practice especially when developing onto something, in this case we are simply using a development tool and it's not a requirement that it runs within a venv, so I don't think you should mention this here

Reply:
My intention is to provide instructions that will work. If I omit installing `pip-compile`, these instructions won't work, and it feels more complicated to try and explain that there are multiple ways I can install pip-compile? If the reader knows how this all works then I think they'll still be okay...

Reply:
no I didn't mean you should omit installing pip-compile, of course. I mean you can omit the suggestion to (re-)create a venv before running the usual pip install something.

Reply:
ooooh yes okay I understand, good point.